Mathlib/RingTheory/MvPolynomial/Homogeneous.lean

```diff
@@ -34,19 +35,34 @@ variable {σ : Type*} {τ : Type*} {R : Type*} {S : Type*}
 
 /-
 TODO
-* create definition for `∑ i in d.support, d i`
 * show that `MvPolynomial σ R ≃ₐ[R] ⨁ i, homogeneousSubmodule σ R i`
 -/
+
+/-- The degree of a monomial. -/
+def degree (d : σ →₀ ℕ) := ∑ i in d.support, d i
+
+theorem degree_eq_weightedDegree (d : σ →₀ ℕ) :
+    ∑ i in d.support, d i = weightedDegree 1 d := by
+  simp [weightedDegree, Finsupp.total, Finsupp.sum]
```

Comment:
I think replacing `∑ i in d.support` with `degree` would make sense, but only if we reverse the direction of this theorem. (Can it be `@[simp]`?)
```suggestion
theorem weightedDegree_one (d : σ →₀ ℕ) :
    weightedDegree 1 d = degree d := by
  simp [weightedDegree, Finsupp.total, Finsupp.sum]
```

Reply:
I have changed this as suggested, but I did not make it a simp lemma, because at least in this file we only use it to replace `degree` by `weigthedDegree` (to recover the proofs from the general case).